src/pages/industries/consulting.jsx

```diff
@@ -0,0 +1,157 @@
+  name: 'Anton Kuliashov',
+  role: 'Software Engineer, Palark',
+  logo: PalarkLogo,
+  CTAtext: 'READ THE BLOG POST',
+  url: 'https://blog.palark.com/why-cilium-for-kubernetes-networking/',
+};
+
+const bulletSection1 = {
+  heading: "Easily Integrate With Customer's  Infrastructure.",
+  paragraphs: [
+    "Consulting companies cater to various clients with different choices of infrastructures and cloud providers. Operating across different environments often results in additional operational and resource overhead. Cilium provides a consistent experience for operators across different environments. With Cilium's cluster mesh,  multiple clusters can be joined into a large unified network regardless of the Kubernetes distribution or location each is running. ",
+    "Cilium also reduces the number of tools operators have to deal with; for example, Cilium's load balancing can replace traditional hardware load balancers. Cilium's Layer 2 announcement can eliminate the need for extra tooling like MetalLB. Together, these features ensure that consulting companies can offer uniform, high-quality services regardless of their client's chosen infrastructure.",
+  ],
+  withImage: true,
+  imageSrc: ConsultingImage1,
+  imageAlt: 'cilium mesh architecture illustration',
+};
+
+const bulletSection2 = {
+  heading:
+    'Visibility Into Customers Infrastructure Across Clouds, Clusters, and Premises, Independent of The Underlying Platform',
+  paragraphs: [
+    'Cilium delivers comprehensive visibility across various platforms, whether on-premises, cloud, or clusters. This unified data plane approach allows for consistent monitoring across deployments, aiding in workload migration between clusters and simplifying the onboarding of new infrastructure providers.',
+    'Observability data from Cilium  is made available using modern standards such as Prometheus and can be visualized using powerful tools such as Grafana. Cilium also supports traditional standards such as sFLow and NetFlow.',
+  ],
+  withImage: true,
+  imageSrc: ConsultingImage2,
+  imageAlt: "cilium's hubble illlustation",
+};
+
+const consultingLogos = [
+  GrayedPalarkLogo,
+  GrayedTietoevryLogo,
+  GrayedMobilab,
+  GrayedCengnLogo,
+  GrayedCistecLogo,
+  GrayedEficodeLogo,
+  GrayedInnoqLogo,
+  GrayedLiquidReplyLogo,
+  GrayedNineNavigatorsLogo,
+  GrayedSchubergLogo,
+  GrayedAyedoLogo,
+  GrayedSuperOrbitalLogo,
+];
+
+const consultingResources = [
+  {
+    title: 'Cilium for Kubernetes networking: Why we use it and why we love it',
+    description:
+      'After years of relying on a simple solution, Palark faced a growing demand for advanced features backed by our customers’ needs. Cilium brought the networking in their K8s platform to the next level',
```

Comment:
```suggestion
      'After years of relying on a simple solution, Palark faced a growing demand for advanced features backed by our customers’ needs. Cilium brought the networking in their Kubernetes platform to the next level',
```

Reply:
My bad, this is mixing first person and third person. rewrote this in third person